client/my-sites/checkout/checkout/style.scss

```diff
@@ -1187,118 +1187,3 @@
 		height: 40px;
 	}
 }
-
-/* Payment Methods Test
-======================================= */
-.payment-box__pm-test-wrapper {
-	background: var( --color-surface );
-	padding: 10px 8px 0;
-
-	@include breakpoint( '>660px' ) {
-		padding: 25px 30px 0;
-	}
-}
-
-.payment-box__pm-title {
-	font-size: 16px;
-	margin-bottom: 8px;
-}
-
-.payment-box__pm-wrapper {
-	margin: 0;
-	padding: 0;
-
-	.section-nav-tab {
-		list-style: none;
-		border: 0;
-		margin: 8px 0 0;
-		position: relative;
-		z-index: z-index( 'root', '.payment-box__pm-wrapper .section-nav-tab' );
```

Comment:
Would it not also be worth removing the z-index registered here: https://github.com/Automattic/wp-calypso/blob/4db3529a9bb9c6869e6587dc91a3acbf1ab732eb/client/assets/stylesheets/shared/functions/_z-index.scss#L68

Reply:
Thanks for the catch. I'll take it out.